staging/src/k8s.io/component-base/metrics/prometheus/workqueue/metrics.go

```diff
@@ -93,8 +93,15 @@ var (
 		Help:           "Total number of retries handled by workqueue",
 	}, []string{"name"})
 
+	delayed = k8smetrics.NewGaugeVec(&k8smetrics.GaugeOpts{
+		Subsystem:      WorkQueueSubsystem,
+		Name:           "delayed",
```

Comment:
In terms of naming I would suggest:
- "delayed_items"
- "delayed_queue_depth"
- "delayed_entries"